spec/support/with_a_deployed_instance.rb

```diff
@@ -0,0 +1,18 @@
+require "moku/command/deploy"
+
+module Moku
+  # Deploy an instance. This should only be invoked in the "with a sandbox"
+  # context.
+  RSpec.shared_context "with a deployed instance" do |instance_name, reference|
+    before(:all) do # rubocop:disable RSpec/BeforeAfterAll
+      Moku.invoker.add_command(
+        Command::Deploy.new(
+          user: ENV["USER"],
+          instance_name: instance_name,
+          reference: reference
```

Comment:
It doesn't look like you ever pass a second parameter to this context. I'd wait to allow the parameter until you're actually going to use it as seeing the parameter here but in all the tests just relying on it being nil is a little confusing.